code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -49,7 +49,7 @@
 
 /obj/item/weapon/pinpointer/examine(mob/user)
 	..(user)
-	for(var/obj/machinery/nuclearbomb/bomb in world)
+	for(var/obj/machinery/nuclearbomb/bomb in machines)

```

Comment:
Hm, actually, you're right. I wonder if it'd be worth starting a general machines list. Since it's trawled often enough.